copier/copier.go

```diff
@@ -1164,6 +1164,10 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 						}
 						symlinkTarget = target
 					}
+					if info.Mode()&os.ModeType == os.ModeSocket {
+						logrus.Debugf("Skipping socket %q", info.Name())
```

Comment:
I thought @rhatdan and @vrothberg had settled on a warning after all?

Reply:
Thanks.  The github page showed only the first comment, reverted back to a warning